tmt-app/src/Components/CareerGuide.js

```diff
@@ -1,12 +1,222 @@
-import React from "react";
-import { Container } from "@mui/system";
-import Breadcrumb from "./Breadcrumb";
+import React, { useState, lazy, Suspense } from "react";
+import {
+  Autocomplete,
+  Box,
+  Container,
+  Grid,
+  InputAdornment,
+  TextField,
+  ToggleButton,
+  ToggleButtonGroup,
+  Typography,
+} from "@mui/material";
+import { FlexContainer } from "../Styling/CustomStyling.js";
+import SearchIcon from "@mui/icons-material/Search";
+import ThaiTeaSwiggle2 from "../Images/art/Thai_Tea_Swiggle_2.png";
+const MajorCard = lazy(() => import("./MajorCard.js"));
+const MajorCareerList = lazy(() => import("./MajorCareerList.js"));
+const GuideSidebar = lazy(() => import("./GuideSidebar.js"));
 
 function CareerGuide() {
+  const filters = ["A to Z", "Z to A"];
+  const [selection, setSelection] = useState("Majors");
+  const handleChoice = (event, newSelection) => {
+    if (newSelection !== null) {
+      setSelection(newSelection);
+    }
+  };
+
   return (
-    <Container>
-      <Breadcrumb />
-    </Container>
+    <Suspense fallback={<div>Loading....</div>}>
+      <Grid
+        container
+        sx={{
+          paddingTop: "7.5rem",
+          minHeight: "100vh",
+          width: "100%",
```

Comment:
Nit: Setting a minimum width size could also be something to think of for a mobile friendly experience =D